pkg/health/clusterStatsHandler.go

```diff
@@ -355,6 +356,10 @@ func getStats(myid uint64, filterFunc func(string) bool, allSegMetas []*structs.
 			allIndexCols[indexName][col] = struct{}{}
 			totalCols[col] = struct{}{}
 		}
+		allIndexCols[indexName][config.GetTimeStampKey()] = struct{}{}
```

Comment:
nit: do we want to get the tskey once at the start of this func and then use it to avoid the mulitple calls in the for loop ?

Reply:
addressed in the latest commit.